css/styles.css

```diff
@@ -0,0 +1,94 @@
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+}
```

Comment:
There should already be a `row` class in your `layout.css` file that was provided . If you wanted an `align-items: stretch`, I would create a class for this in your `layout.css` file named

```
.items-stretch {
  align-items: stretch;
}
```